enhancements/network/dpu/ocp-on-dpu.md

```diff
@@ -0,0 +1,260 @@
+    support anywhere for installing the infra and tenant clusters
+    as a unit.
+
+  - Upgrades to the infra cluster will result in network outages in
+    the tenant cluster, as there will be no coordination between
+    infra cluster and tenant cluster reboots.
+
+  - The infra cluster will require separate ARM master nodes. We may
+    support running these as VMs on an x86 host.
+
+  - There will be no support for running customer workloads of any
+    sort in the infra cluster.
+
+  - It will not necessarily be possible to upgrade from Dev Preview
+    to Tech Preview without reinstalling one or both clusters.
+
+- Create further Tech Preview and GA versions of the feature with
+  improved functionality.
+
+### Non-Goals
+
+- ...
+
+## Proposal
+
+### User Stories
+
+See Overview doc.
+
+### Risks and Mitigations
+
+#### Hardware Availability
+
+We are currently developing using pre-GA BlueField-2 hardware, with a
+plan based on official descriptions of how the final GA BlueField-2
+hardware will work. So far (presumably due to the standard global chip
+supply chain problems?) we have been unable to get GA cards (either
+for free or by paying for them). If we are unable to get GA cards
+soon, it may not be possible to ship the Dev Preview as designed for
+OCP 4.10.
+
+#### OCP on ARM
+
+Some parts of OCP-on-bare-metal are not currently available for ARM.
+In particular, I think we don't have "ironic" images, which are needed
+for an IPI install.
+
+## Design Details
+
+### Design Details - Dev Preview
+
+The initial dev preview will be a very limited implementation, to
+demonstrate the overall architecture.
+
+#### Installation
+
+Installing a cluster with BF-2 cards is potentially tricky, since the
+cards can be configured in multiple modes, and in some modes, may not
+allow their host to access the network by default.
+
+To install the cluster then, it will be necessary to:
+
+- Install the tenant cluster masters and at least one non-BF-2 worker
+  node.
+
+- Create a MachineConfigPool in the tenant cluster for the BF-2
+  workers, and update the Cluster Network Operator configuration to
+  indicate that this pool is for BF-2 workers.
+
+- Power on, but don't yet install, the BF-2 hosts in the tenant
+  cluster (so that the BF-2s themselves will be powered on).
+
+- Install the infra cluster.
+
+- Install the [DPU Operator] onto the infra cluster, providing it with
+  an appropriate tenant-cluster kubeconfig. Initially, this will just
+  ensure that each of the cards is configured in "passthrough" mode,
+  allowing the tenant hosts unrestricted network access.
+
+- Add the BF-2 workers to the tenant cluster. The combination of the
+  CNO config in the tenant cluster and the DPU Operator in the infra
+  cluster will cause each BF-2 worker to be configured to run
+  ovn-kubernetes in "DPU Host" mode with its corresponding NIC running
+  ovn-kubernetes in "DPU NIC" mode.
+
+[DPU Operator]: https://github.com/openshift/enhancements/pull/890
```

Comment:
We have the link to DPU Operator in the below `DPU Opeator` section. Maybe remove this or simply refers to the latter.